00-RELEASENOTES

```diff
@@ -9,8 +9,8 @@ Usually "unstable" is stable enough for you to use it in development environment
 however you should never use it in production environments. It is possible
 to download the latest stable release here:
 
-    https://download.redis.io/redis-stable.tar.gz
+    https://github.com/valkey-io/valkey/releases
```

Comment:
```suggestion
    https://valkey.io/download/
```
If we want to keep the same intention.

Reply:
It's not necessarily a permalink to stability, but it will include the latest stable releases.

Reply:
it makes sense, will update!